packages/serverless-deploy-iam/bin/app.ts

```diff
@@ -148,6 +148,7 @@ export class ServiceDeployIAM extends cdk.Stack {
             "lambda:ListEventSourceMappings",
             "lambda:CreateEventSourceMapping",
             "lambda:DeleteEventSourceMapping",
+            "lambda:TagResource",
```

Comment:
Why do we need this on all lambdas? Can we just scope this to named lambdas? (The one above?)

We probably also need the `UntagResource` permission too

Reply:
Thanks for picking this up. Yep, wrong approach. Closing this PR off.